schema/codelists/relatedProcess.csv

```diff
@@ -1,9 +1,10 @@
-Code,Title,Description
-framework,Framework agreement procedure first stage,This contracting process follows on from the related process to establish a framework. The suppliers on the framework are listed in the award section of the related process.
-planning,Planning process,This contracting process follows on from the related planning process.
-parent,Parent contract (for sub-contracts),This contracting process might result in a sub-contract of the related process.
-prior,Prior process,This contracting process is the renewal or replacement of the related prior process.
-unsuccessfulProcess,Unsuccessful process,This contracting process follows on from an previous unsuccessful process.
-subContract,Sub-contract,The related process might result in a sub-contract of this contract.
-replacementProcess,Replacement process,The related process might result in the replacement of this contract.
-renewalProcess,Renewal process,The related process might result in the renewal of this contract.
+Code,Title,Description,Deprecated,Deprecation note
+framework,Framework agreement procedure first stage,The contracting process is the second stage of a framework agreement procedure. The related process is the contracting process that established the framework agreement.,,
+planning,Planning process,The related process is the planning process that planned the contracting process.,,
+parent,Parent contract (for sub-contracts),The contracting process might result in a sub-contract of the related contracting process.,,
+prior,Prior process,The related process is a contracting process that concluded related contracts.,1.2,Deprecated in favor of 'successfulProcess'.
+successfulProcess,Successful process,"The contracting process aims to extend, renew, repeat or replace one or more contracts from the related contracting process.",,
+unsuccessfulProcess,Unsuccessful process,The contracting process is another attempt to conclude one or more contracts. The related process is the contracting process that attempted and failed to conclude the contracts previously.,,
+subContract,Sub-contract,The related process might result in a sub-contract of this contract.,1.2,Deprecated in favor of 'parent' because OCDS 1.2 follows a child refers to parent approach to linking related processes.
+replacementProcess,Replacement process,The related process might result in the replacement of this contract.,1.2,Deprecated in favor of 'successfulProcess' because  OCDS 1.2 follows a child refers to parent approach to linking related processes.
+renewalProcess,Renewal process,The related process might result in the renewal of this contract.,1.2,Deprecated in favor of 'successfulProcess' because  OCDS 1.2 follows a child refers to parent approach to linking related processes.
```

Comment:
Repeat same change elsewhere.

```suggestion
renewalProcess,Renewal process,The related process might result in the renewal of this contract.,1.2,Deprecated in favor of 'successfulProcess', because OCDS 1.2 follows a ""child refers to parent"" approach to linking related processes.
```

Reply:
Done in 31a4807be425224afa749ac191d946ccd3f0f90a